ios/FluentUI.Demo/FluentUI.Demo/DemoColorThemes.swift

```diff
@@ -208,4 +213,9 @@ class DemoColorGreenTheme: NSObject, ColorProviding {
         return UIColor(dynamicColor: DynamicColor(light: ColorValue(0x0F703B),
                                                   dark: ColorValue(0x60BD82)))
     }
+
+    var brandStrokeTint: UIColor {
```

Comment:
`brandStrokeTint` for the green theme has not been defined yet so as a placeholder, I used the values for `brandStroke1`. Currently waiting on design

Reply:
Idea for future improvement: we should probably put all these hex values in a table using the traiditional `brand10` through `brand160` naming, to make it easier to see how they're being mapped in the theme.

Reply:
Based off offline conversations, since adding `brandStrokeTint` will be a breaking change and it has not been defined for the app ramps, we'll it in a subsequent PR.